### ISSUE TITLE
Sound Quality Drop and Not Loading Project

### ISSUE THREAD

**Issue by Roemmato** [Mittagskogel/Sulfurous#44]
The program has been working wonderfully with my project, everything is running smoothly. Until I upload a few voice clips. The other voice clips I uploaded were fine and nice and they sounded wonderful, but this one, it sounded distorted, pixelated it was messed up and didn't even sound like the person who voiced them. After that, the project would stop loading if I dragged the SB2 into the project, but if I put a link it would be fine.

Here's the clip I was talking about, so I don't know if it's me or just a problem with the program https://mega.nz/#!M8RWFJAD

Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/59.0.3071.115 Safari/537.36

**Comment by htmlgames** [comment]
Note that any wav file over 30 seconds is stored in a different (mono) format in scratch. This may be the issue.

Also,iIt could be that this wav was encoded using a different format (wav is just a container for pcm sounds).
If you have access to a sound recorder program (or sndrec32.exe for windows) you should be able to get the properties of that file against the ones that sound better, and check this.

**Comment by Roemmato** [comment]
@htmlgames 
The file I sent was a mp3. I cut up the sounds because of how the game works they needed to be said at different times.

**Comment by htmlgames** [comment]
Ok, so what I would do is use a mp3 to wav converter program, try a couple of mp3 cuts to wav and upload them to scratch and make sure the quality is ok. (rather than letting scratch do the conversion)

Once you have a wav format your happy with (the converter may offer to save wav in different formats) AND the length of each cut is less than 30 secs, I don't see how any distortion could occur.

If any of the cuts are > 30 secs then you can get sound degradation.

**Comment by Roemmato** [comment]
@htmlgames This method worked, although I didn't do the wav it worked pretty well. My only problem is dragging my SB2 in Sulfurous. Your program did work to load in my project though, so thanks for the help! =)

**Comment by htmlgames** [comment]
Ok, so are you happy now with your project in Sulfurous?

**Comment by Roemmato** [comment]
Yes, the quality of the sound is much better. My only problem is dragging the SB2 into the projects which for some reason won't work, I have tried uploading it to scratch then putting in the URL which worked, but dragging it simply won't work.

**Comment by htmlgames** [comment]
Ah, I see - so the dragging of the project onto the htmlgames version works, but not in the sulfurous version.

The htmlgames version doesn't have as good svg rendering as you stated.

The only thing now to suggest (as I don't have any ref to the sb2 project) is for you to open sulfurous in your web browser,, press F12 (to get the debugger) and switch to the console, drop you file on the sulfurous window as before as report any errors in the console for me to work with.

Also if its a big sb2 file it may take quite a while to parse...

Regards,

**Comment by Roemmato** [comment]
@htmlgames Ah, that may be the problem, the SB2 is 45.3 MB =/

**Comment by Mittagskogel** [comment]
Sound should work correctly with all files now. I'm not sure if the 45.3MB would be a problem, you might just need to wait some time for all of the assets to get processed...
